src/test/java/com/example/project/page_object/ShoppingBagPage.java

```diff
@@ -0,0 +1,26 @@
+package com.example.project.page_object;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class ShoppingBagPage {
+    private WebDriver driver;
+
+    @FindBy(css="h3.qa-cart-item-name")
```

Comment:
In case of css locators that could simply be` @FindBy("h3.qa-cart-item-name")`

Reply:
hm, it doesn't seem to work